### ISSUE TITLE
[Bug] Results are vastly different after converting a PyTorch model to IR (conversion problem?)

### ISSUE THREAD

**Issue by runin27** [openvinotoolkit/openvino#11512]
Hello, I am trying to develop a plant disease detection system through transfer learning. The pre-trained model that I'm using is **ShuffleNetv2 x1.0**. It was successfully modified and trained through **PyTorch** and has made successful inferences with an accuracy of 90+%. However, after converting it to **ONNX** and then to **IR**, it's been giving me entirely different results when using **OpenVINO**. Pretty much 0% accuracy.

Referencing from this post: [the inference result is totally different after converting onnx to openvino IR](https://community.intel.com/t5/Intel-Distribution-of-OpenVINO/the-inference-result-is-totally-different-after-converting-onnx/td-p/1135937) 
I have checked and tried every method available and normalised all inputs. I was wondering if there's anything wrong with the conversion parts, either from **.pth to .onnx**, or **.onnx to IR**.

Attached is my ShuffleNet model: [ShuffleNet.zip](https://github.com/openvinotoolkit/openvino/files/8479255/ShuffleNet.zip)

This is my conversion process from .pth to .onnx: (some libraries and test transformation not included)
````
trained_model = torch.load('/shfnetmodel.pth')

import torch
from torch.autograd import Variable

dummy_transform = image_transforms['test']
dummy_input = dummy_transform(Image.open('/content/output/MyDrive/outputdata/test/rice_unhealthy/rice_d('+str(15)+').jpg')).view(1, 3, 224, 224).cuda()

torch.onnx.export(trained_model, dummy_input, "/shfnetmodel.onnx")
````

This is my conversion from .onnx to IR:
`mo_onnx.py --input_model "D:\...\shfnetmodel.onnx" --output_dir "D:\...\ShuffleNet” --data_type FP32 --mean_values [123.675,116.28,103.53]  --scale_values [58.395,57.12,57.375] --reverse_input_channels
`

**Comment by jiwaszki** [comment]
Hi @runin27 ,

Did you try running inference on ONNX model itself? Were your results also around 0%?

I also followed the forum thread attached by you. Are you sure that ` --reverse_input_channels` applies to your case?
Moreover most of users on forum use simple `x=torch.randn((1,3,224,224))` for dummy inputs. Is there a specific reason that transforms need to be applied?

**Comment by runin27** [comment]
Hello,

I just tried running inference on ONNX model itself and it works well! 

I thought that the transforms needed to be applied to the dummy input as well to get the same parameters. But I have tried using just `x=torch.randn((1,3,224,224))` and have confirmed that they pretty much work the same.

I have tried converting the ONNX model to xml and bin without the `--reverse_input_channels`, and it still did not work, confirming that the conversion from .onnx to IR is the issue here. but I cant figure out which part of the conversion I did wrong.

**Comment by mateusztabaka** [comment]
Have you tried with just `mo_onnx.py --input_model "D:\...\shfnetmodel.onnx" --output_dir "D:\...\ShuffleNet” ` (so without mean_values, scale_values and reverse_input_channels) and then running inference on D:\...\ShuffleNet\shfnetmodel.xml model?

**Comment by runin27** [comment]
I think I have tried this with different combinations of conversions in the past and forgot to try this again. I just did and it is working fine now!! Thank you so much @mateusztabaka and @jiwaszki
